soapclient/SforceEnterpriseClient.php

```diff
@@ -25,7 +25,6 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 require_once ('SforceBaseClient.php');
-

```

Comment:
The whitespace here and on other lines does not need to be removed. It is usually preferred to have an empty line between blocks to improve readability.